src/gui/optionsdialog.ui

```diff
@@ -616,6 +637,9 @@
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QCheckBox" name="checkFileLogDelete">
+                   <property name="toolTip">
+                    <string>Deletes backup logs older than specified time</string>
```

Comment:
I think the option is descriptive enough.

Reply:
Removed